coding-102-parsers/strings.py

```diff
@@ -8,4 +8,143 @@
 
 '''
 
-import re
\ No newline at end of file
+import re
+
+STRING = '''
+ Neighbor: 21.0.0.1, Address-Family: ipv4 Unicast (VRF1)
+   Locally configured policies:
+    route-map RMAP in
+    route-map RMAP2 out
+ Neighbor: 21.0.0.2, Address-Family: ipv6 Unicast (VRF2)
+   Locally configured policies:
+    route-map RMAP3 in
+    route-map RMAP4 out
+ Neighbor: 21.0.0.3, Address-Family: VPNv4 Unicast
+   Locally configured policies:
+    route-map RMAP5 in
+    route-map RMAP6 out
+ Neighbor: 21.0.0.4, Address-Family: VPNv6 Unicast (VRF2)
+   Locally configured policies:
+    route-map RMAP7 in
+    route-map RMAP8 out
+    '''
+
+out = STRING
```

Comment:
don't need to assign out to STRING
just do for line in STRING.splitlines()

Reply:
Done.